google/cloud/bigtable/data/read_rows_query.py

```diff
@@ -48,11 +53,24 @@ def __init__(
         start_is_inclusive: bool | None = None,
         end_is_inclusive: bool | None = None,
     ):
+        """
+        Args:
+          - start_key: The start key of the range. If None, the range is unbounded on the left.
+          - end_key: The end key of the range. If None, the range is unbounded on the right.
+          - start_is_inclusive: Whether the start key is inclusive. If None, the start key is
+                inclusive.
```

Comment:
Why not set the default value to True? Instead of default being None and then using an if?

Reply:
This way lets us know if they explicitly passed in a value. If we used a default, we won't know if they set it True manually, or if it's falling back to the default.

I think the reason we did this here was because it doesn't really make sense to set the inclusive flag when the key is None, so we raise an error. This was the behaviour of the existing [RowFilter class](https://github.com/googleapis/python-bigtable/blob/62dd21798a5c646fb5628e47d484cfb8a594de50/google/cloud/bigtable/row_filters.py#L375)

Reply:
I dont know if I agree, but I dont want to block this pr. We can always change it

Reply:
Yeah I'm not too attached to either way. I opened a bug for future discussion: https://github.com/googleapis/python-bigtable/issues/826